pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -14,14 +15,16 @@ import (
 
 // StorageClusterSpec defines the desired state of StorageCluster
 type StorageClusterSpec struct {
-	ManageNodes  bool   `json:"manageNodes,omitempty"`
-	InstanceType string `json:"instanceType,omitempty"`
+	ManageNodes   bool                 `json:"manageNodes,omitempty"`
+	InstanceType  string               `json:"instanceType,omitempty"`
+	LabelSelector metav1.LabelSelector `json:"labelSelector,omitempty"`
 	// External Storage is optional and defaults to false. When set to true, OCS will
 	// connect to an external OCS Storage Cluster instead of provisioning one locally.
 	ExternalStorage ExternalStorageClusterSpec `json:"externalStorage,omitempty"`
 	// HostNetwork defaults to false
 	HostNetwork bool `json:"hostNetwork,omitempty"`
 	// Resources follows the conventions of and is mapped to CephCluster.Spec.Resources
+	Placement          rookv1.PlacementSpec                   `json:"placement,omitempty"`
```

Comment:
What is this used for?

Reply:
Expose placements to specify placements for OCS on unlabelled nodes / custom label nodes.